src/ngraph/pass/core_fusion.cpp

```diff
@@ -202,15 +202,15 @@ void pass::CoreFusion::construct_folded_batch_norm()
     auto beta = std::make_shared<pattern::op::Label>(element::f32, beta_shape);
     double eps = 0.001;
     auto shape_r = Shape{1, 2, 2, 2};
-    auto bn = std::make_shared<op::BatchNorm>(eps, gamma, beta, pconv, mean, var);
+    auto bn = std::make_shared<op::BatchNormInference>(eps, gamma, beta, pconv, mean, var);
```

Comment:
does the batch norm fording only applies to `Inference` Op?. I think  we can apply the below transformation during training as well.
```
        // new weights = old weights * gamma / sqrt(variance + epsilon)
        // new biases = -mean * gamma / sqrt(variance + epsilon) + beta
```